src/test/java/ua/softserveinc/tc/config/TestBaseConfigClass.java

```diff
@@ -21,7 +21,7 @@
     private static final String PROPERTY_NAME_DATABASE_DRIVER = "com.mysql.jdbc.Driver";
     private static final String PROPERTY_NAME_DATABASE_URL = "jdbc:mysql://localhost:3306/kidsroom_test";
     private static final String PROPERTY_NAME_DATABASE_USERNAME = "root";
-    private static final String PROPERTY_NAME_DATABASE_PASSWORD = "root";
+    private static final String PROPERTY_NAME_DATABASE_PASSWORD = "melancholiya20";
```

Comment:
Why you want did this? I think that leave simple password such as "root" is best choice.